modules/dataquery/README.md

```diff
@@ -70,7 +70,8 @@ curl -H 'Content-Type: application/json' -X POST http://$YOURCOUCHDBADMIN:$YOURC
 
 ###### Import the LORIS instrument data
 
-This step is optional and not required if only the MRI portion of LORIS is used:
+This step is optional and not required if only the MRI portion of LORIS is used: \
```

Comment:
I would prefer to indicate when it is required. Other options might be invalidated in the future.
Similar change should also be applied to the MRI section below.
```suggestion
This step is optional and is required only if instruments are used: \
```

Reply:
I believe the suggestion is not the original meaning, and contradictory.  @PapillonMcGill

Reply:
@lingma you are right that it is not the original meaning, but @PapillonMcGill 's change is also more accurate and so better

Reply:
however not relevant to this PR which I will close.